tools/aa/commands/WorkspaceCommand.java

```diff
@@ -88,10 +91,13 @@ private boolean createWorkspace() {
     } else {
       fileUtils.mkdirs(workspacePath);
       try {
+        List<String> filesAndFoldersToIgnore = new ArrayList<>();
+        filesAndFoldersToIgnore.add("^bazel-.*$");
+        filesAndFoldersToIgnore.addAll(getFilesOrFoldersToIgnoreInHead("node_modules"));
```

Comment:
What if we do a regex?
Try:
`node_modules`
`^node_modules$`
Not sure if we're matching against the full path or the folder name.

Reply:
Added regex support.